frontend/packages/citizen-frontend/src/localization/en.ts

```diff
@@ -236,63 +236,66 @@ const en: Translations = {
           }
         },
         contactInfo: {
-          title: 'Henkilötiedot',
+          title: 'Personal information',
           child: {
-            title: 'Lapsen tiedot',
-            name: 'Lapsen nimi',
-            ssn: 'Lapsen henkilötunnus',
-            streetAddress: 'Kotiosoite',
-            isAddressChanging: 'Osoite muuttunut / muuttumassa',
+            title: 'Child´s information',
```

Comment:
Incorrect punctuation, should be:
```suggestion
            title: "Child’s information",
```

Reply:
Seems like you missed this one